SECP-service/src/main/java/com/visucius/secp/Controllers/User/UserProfileController.java

```diff
@@ -0,0 +1,105 @@
+package com.visucius.secp.Controllers.User;
+
+
+import com.google.common.base.Optional;
+import com.visucius.secp.DTO.UserDTO;
+import com.visucius.secp.models.User;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.visucius.secp.daos.UserDAO;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+
+public class UserProfileController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(UserProfileController.class);
+
+    private final UserDAO userDAO;
+
+    public UserProfileController(UserDAO userDAO){
+        this.userDAO = userDAO;
+    }
+
+    public Response getProfile(String userID){
+        long id = Long.parseLong(userID);
```

Comment:
there would be an error here if i pass in "abc" as userID. Take a look at  other controllers to see how i handled this.